scriptoocloud/src/main/java/edu/asu/diging/scriptoocloud/web/CloneRepositoryController.java

```diff
@@ -0,0 +1,57 @@
+package edu.asu.diging.scriptoocloud.web;
+
+import javax.validation.Valid;
+
+import org.eclipse.jgit.api.errors.GitAPIException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import edu.asu.diging.scriptoocloud.core.forms.CloneForm;
+import edu.asu.diging.scriptoocloud.core.service.CloneRepository;
+
+@Controller
+public class CloneRepositoryController {
+
+    @Autowired
+    CloneRepository cloneService;
+ 
+    @RequestMapping(value = "/listrepos", method = RequestMethod.GET)
+    public String listRepos(Model model) {
+        model.addAttribute("repo",cloneService.getRepos());
+        return  "github/ListRepos";
+    }
+
+    @RequestMapping(value = "/listrepos/{host}/{owner}/{repo}/{requester}", method = RequestMethod.POST)
```

Comment:
This method is for deleting repositories already added, right? you should use their id to delete them. Also, this method should be in its own controller.